discovery_utils/getters/hansard.py

```diff
@@ -0,0 +1,145 @@
+import json
+import logging
+import os
+import xml.etree.ElementTree as ET
+
+from io import BytesIO
+from pathlib import Path
+
+import boto3
+import pandas as pd
+
+from botocore.client import BaseClient
+from botocore.exceptions import ClientError
+from dotenv import load_dotenv
+
+from discovery_utils import PROJECT_DIR
+from discovery_utils.utils import s3 as s3
+
+
+load_dotenv()
+
+# Retrieve AWS file information from environment variables
+S3_BUCKET = os.getenv("S3_BUCKET")
+# S3_PATH_RAW = os.getenv("S3_PATH_RAW")
+# FILE_NAMES_RAW = json.loads(os.getenv("FILE_NAMES_RAW"))
+
+# Retrieve AWS credentials from environment variables
+AWS_ACCESS_KEY = os.getenv("AWS_ACCESS_KEY")
+AWS_SECRET_KEY = os.getenv("AWS_SECRET_KEY")
+
+# Configure logging
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+
+class HansardGetter:
+    def __init__(self):
+        self.s3_client = s3.s3_client()
+        self.bucket = S3_BUCKET
+        self.s3_prefix = "data/policy_scanning_data/"
+        logger.info(f"Initialized HansardGetter with bucket: {self.bucket}")
+
+    def s3_client(aws_access_key_id: str = AWS_ACCESS_KEY, aws_secret_access_key: str = AWS_SECRET_KEY) -> BaseClient:
+        """Initialize S3 client"""
+        S3 = boto3.client("s3", aws_access_key_id, aws_secret_access_key)
+        return S3
```

Comment:
Hmm, a bit confused if you're using this function or s3_client function from s3 module imported above to initialise?